### PR TITLE
Fix #746: Nothing happens if user clicks on Transaction History

### LINKED ISSUE #746: Nothing happens if user clicks on Transaction History
**Summary:**

Nothing happens if user clicks on Transaction History on Home Fragment

**Steps to reproduce:**

1. Navigate to Home Fragment
2. Swipe up Transaction History
3. Click on any transaction
4. Nothing happens

**Expected behavior:**

Ideally, if user clicks on any transaction, app should show transaction details as done in History Fragment

**Observed behavior:**

Nothing happens after clicking any transaction

**Device and Android version:**

Android 10
OnePlus 6
Stock ROM

**Screenshots:**

mobile-wallet - nothing happens after transaction history

### PR DESCRIPTION
**Pull request by Grandolf49**
## Issue Fix
Fixes #746 If user clicks on transaction under transaction history on Home Fragment, app will show the transaction details

## Screenshots

fix - nothing happens after transaction history

##

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by Grandolf49** [comment]
@naman14 What are your thoughts on this PR?

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/presenter/HomePresenter.java [hunk 1]
```diff
@@ -117,4 +120,15 @@ public void showMoreHistory(int existingItemCount) {
             handleTransactionsHistory(existingItemCount);
         }
     }
+
+    @Override
+    public void fetchTransactions() {
```
Comment (rachittt) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/presenter/HomePresenter.java:125]: @Grandolf49 remove this empty function.

Comment (Grandolf49) [reply mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/presenter/HomePresenter.java:125]: Hi @rachittt, Thanks for pointing out. I'll work that out.

**Comment by shiv07tiwari** [comment]
@Grandolf49 This will make this section similar to the history fragment. This screen is basically to have a quick look into the transaction history.

**Pull request closed without merging**